docs/en/observability/monitor-k8s/monitor-k8s-add-integration.asciidoc

```diff
@@ -0,0 +1,325 @@
+
+[discrete]
+[[monitor-kubernetes-integration-container-logs]]
+=== Collect Kubernetes container logs
+
+Turn this option on to collect and parse Kubernetes container logs.
+Containers running within Kubernetes pods publish logs to stdout or stderr.
+These logs are written to a location known to kubelet.
+By default, the container parser is enabled.
+You can enable additional parsers in *advanced settings*.
+
+Refer to {integrations-docs}/kubernetes/container-logs[Kubernetes container logs] for more on collecting container logs.
+
+[discrete]
+[[monitor-k8s-container-log-configure]]
+==== Configure Kubernetes container logs
+
+Provide the following information to collect container logs:
+
+[horizontal]
+*Use Symlinks*:: A symlink is lightweight and doesn't contain the data of the log files, but points to their actual location. Turn this on to use symlinks.
+*Condition*:: You can specify a condition to control whether a configuration is applied to the running Elastic Agent.
+
+[discrete]
+[[monitor-kubernetes-integration-audit-logs]]
+=== Collect Kubernetes audit logs
+
+preview::[]
+
+Turn this option on to collect audit logs.
+Kubernetes audit logs record requests that come to the Kubernetes API from internal and external components.
+These logs help you understand cluster behavior and debug issues.
+
+Refer to {integrations-docs}/kubernetes/audit-logs[Kubernetes audit logs] for more on collecting audit logs.
+
+[discrete]
+[[monitor-k8s-update-agent-config]]
+== Part 2: Configure and install the standalone {agent}
+
+After configuring your integration, you need to download and update your manifest. First, download the manifest by  completing the following steps:
+
+. At the bottom of the *Add Kubernetes integration* page, click *Save and continue*.
+. Click *Add {agent} to your hosts*.
+. Under *Enroll in Fleet?*, select *Run standalone*.
+. Under *Configure the agent*, click *Download Manifest*.
+
+After downloading the manifest, update the `ES_USERNAME` and `ES_PASSWORD` environment variables in the DaemonSet to match your {es} credentials.
+
+You can also further modify the manifest to fit your needs. For example, you might want to enable autodiscovery to automatically discover container logs. Refer to the {fleet-guide}/elastic-agent-kubernetes-autodiscovery.html[autodiscovery docs] in the {fleet} guide for more on enabling autodiscovery in your manifest.
+
+Once you are ready to deploy your {agent}:
+
+. From the directory where you've downloaded the manifest, run the following apply command:
++
+["source", "sh", subs="attributes"]
+----
+kubectl apply -f elastic-agent-standalone-kubernetes.yml
+----
+. Check the {agent} status with the following command:
++
+["source", "sh", subs="attributes"]
+----
+kubectl -n kube-system get pods -l app=elastic-agent
+----
```

Comment:
Refer to https://www.elastic.co/guide/en/fleet/current/debug-standalone-agents.html to debug elastic agent standalone